File: PokerDice/src/Die.java
public class Die {
	
	private int faceValue;
	
	public Die() {
		faceValue = 1;
	}
	
	public int roll() {
		faceValue = (int)(Math.random() * 6) + 1;
		return faceValue;
	}

	public void setFaceValue (int value) {
		faceValue = value;
	}

	public int getFaceValue() {
		return faceValue;
	}

	public String toString() {
		String result = Integer.toString(faceValue);
		return result;
		}
}

File: PokerDicePython/Die.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


class Die(object):
    """ generated source for Die

    """
    faceValue = 0

    def __init__(self):
        self.faceValue = 1

    def roll(self):
        self.faceValue = Math.random() * 6 + 1
        return self.faceValue

    def setFaceValue(self, value):
        self.faceValue = value

    def getFaceValue(self):
        return self.faceValue

    def toString(self):
        result = Integer.toString(self.faceValue)
        return result


